packages/manager/apps/pci-instances/src/pages/instances/create/components/flavor/FlavorSelection.component.tsx

```diff
@@ -48,8 +48,9 @@ export const FlavorSelection: FC<{ withUnavailable: boolean }> = ({
   const { trackClick } = useOvhTracking();
 
   const flavorsData = useMemo(
-    () => selectFlavors(deps)(projectId, flavorType, microRegion),
-    [flavorType, microRegion, projectId],
+    () =>
+      selectFlavors(deps)(projectId, flavorType, microRegion, withUnavailable),
```

Comment:
We could prevent from having so much arguments on the curried function by using a business object :)


```suggestion
      selectFlavors(deps)({projectId, flavorType, microRegion, withUnavailable}),
```